src/OpenTelemetry.Api/Trace/TracerProviderBuilder.cs

```diff
@@ -14,6 +14,7 @@
 // limitations under the License.
 // </copyright>
 using System;
+using System.Diagnostics;
```

Comment:
Without this, the project fails to build due to the <see cref="Activity"> XML comment below on AddLegacySource. Wondering if this is enough of a drain to prefer instead rewriting the below comment?

Reply:
if you want, you could move this to the comment. So, `System.Diagnostics.Activity`

Reply:
This shouldn't affect the runtime performance of the application, right? We should add that using directive so that the method documentation is written with brevity.

Reply:
Dont think there is any issue here. My personal preference is to not add the using Sys.Diag, and instead use the full name in the doc comment. It'll make using look cleaner.

Reply:
Merging without this - you can address this in a separate PR.